src/Widgets/DiskGrid.vala

```diff
@@ -19,21 +19,35 @@
  */
 
 public class Installer.DiskButton : Gtk.ToggleButton {
-    public Disk disk { get; construct; }
-    public DiskButton (Disk disk) {
+    public unowned Distinst.Disk disk { get; construct; }
+    public DiskButton (Distinst.Disk disk) {
         Object (disk: disk);
     }
 
     construct {
         margin = 12;
         get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
 
-        var disk_image = new Gtk.Image.from_icon_name (disk.get_icon_name (), Gtk.IconSize.DIALOG);
+        // Drives are identifiable by whether they are rotational and/or removable.
+        string icon_name = null;
+        if (disk.is_removable ()) {
+            if (disk.is_rotational ()) {
+                icon_name = "drive-harddisk-usb";
+            } else {
+                icon_name = "drive-removable-media";
```

Comment:
Probably want to add `-usb` to this. In Pop! they're the same, but in elementary they are distinct, and I think the USB would be more appropriate.